stdlib/public/core/CompilerProtocols.swift

```diff
@@ -338,7 +338,7 @@ public protocol ExpressibleByUnicodeScalarLiteral {
   /// A type that represents a Unicode scalar literal.
   ///
   /// Valid types for `UnicodeScalarLiteralType` are `UnicodeScalar`,
-  /// `String`, and `StaticString`.
+  /// `Character`, `String`, and `StaticString`.
```

Comment:
I believe this is now covered by https://github.com/apple/swift/pull/7125/files/2201e882fa4715afb2b941bc86ed3ef991cc5eae..e67a2b3439b27d2d31d4b6a9745cbf1664002ed0 thanks for the feedback :)